src/slider.jsx

```diff
@@ -307,6 +307,15 @@ const Slider = React.createClass({
   },
 
 
+  _toggleSelection(value) {
```

Comment:
Maybe, we could add a comment, saying it's needed to prevent text selection, and that in the future, we should use an `input` type range.